snippets/go/clear_payload.go

```diff
@@ -0,0 +1,19 @@
+package client
+
+import (
+	"context"
+
+	"github.com/qdrant/go-client/qdrant"
+)
+
+func clearPayload() {
+	client, _ := qdrant.NewClient(&qdrant.Config{
+		Host: "localhost",
+		Port: 6334,
+	})
+
+	client.ClearPayload(context.Background(), &qdrant.ClearPayloadPoints{
```

Comment:
same here

Reply:
I thought of checking the error.
But ignored it to keep the snippet a little concise.